web/pgadmin/static/js/SchemaView/styled.jsx

```diff
@@ -0,0 +1,170 @@
+/////////////////////////////////////////////////////////////
```

Comment:
Please rename this to `Styled` or `StyledComponents` since it includes react components.

Reply:
Done.